views/forms/BarChartFields.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from 'react'
+import Delete from '@material-ui/icons/Delete';
+import TextField from '@material-ui/core/TextField'
+import Button from '@material-ui/core/Button'
+import Typography from '@material-ui/core/Typography';
+
+const title = 'title'
+
+const BarChartFields = ({ 
+  classes,
+  updateFormValues,
+  formValues,
+  fieldLabelValueMap,
+  addValueAndLabelField,
+  removeField,
+  updateFieldValues
+}) => {
+  const [chartTitle, setTitle] = useState('')
+  
+  useEffect(() => {
+    setTitle(formValues[title])
```

Comment:
@mikestone14  It's used on this line as the property to set the title
https://github.com/AMP-SCZ/dpdash/pull/84/files#r910285233

Reply:
hmm, I think there might be a more straightforward way to get this functionality since we don't need to depend on all form values of the label value map

```jsx
const { title } = formValues

useEffect(() => {
  setChartTitle(title || '')
}, [title])

Reply:
[78b1245](https://github.com/AMP-SCZ/dpdash/pull/84/commits/78b12458361f85aeb0919c885be4ca8c8f6c047b)